<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newShader</class>
 <widget class="QWidget" name="newShader">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New layer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Property</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Display</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="display">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
          </property>
          <item>
           <property name="text">
            <string>default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>alpha</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>add</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>none</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="type">
          <item>
           <property name="text">
            <string>rgba</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rgba32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rgb</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>luminance</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="ID">
          <property name="maximum">
           <number>4096</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="width">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>8192</number>
            </property>
            <property name="value">
             <number>1280</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="height">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>8192</number>
            </property>
            <property name="value">
             <number>720</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="resolutionCheck">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Resolution</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="backBuffer">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Back buffer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Channels</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="chan1LineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="chan2LineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLineEdit" name="chan3LineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="chan4LineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="chan1ImageButton">
          <property name="text">
           <string>Image...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="chan2ImageButton">
          <property name="text">
           <string>Image...</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="chan3ImageButton">
          <property name="text">
           <string>Image...</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QPushButton" name="chan4ImageButton">
          <property name="text">
           <string>Image...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="chan1PassButton">
          <property name="text">
           <string>Layer...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QPushButton" name="chan2PassButton">
          <property name="text">
           <string>Layer...</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QPushButton" name="chan3PassButton">
          <property name="text">
           <string>Layer...</string>
          </property>
         </widget>
        </item>
        <item row="6" column="5">
         <widget class="QPushButton" name="chan4PassButton">
          <property name="text">
           <string>Layer...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="chan1Name"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="chan2Name"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="chan3Name"/>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="chan4Name"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>resolutionCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>width</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resolutionCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>height</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
